install/kubernetes/templates/istio-ingress.yaml.tmpl

```diff
@@ -42,7 +42,7 @@ spec:
       serviceAccountName: istio-ingress-service-account
       containers:
       - name: istio-ingress
-        image: {PROXY_HUB}/proxy_debug:{PROXY_TAG}
+        image: {PROXY_HUB}/proxy:{PROXY_TAG}
```

Comment:
it should be a variable so I can have debug images if I want ?
(applies everywhere/below)